### ISSUE TITLE
Is project still active?

### ISSUE THREAD

**Issue by lehn-etracker** [express42/zabbixapi#88]
I'm asking myself if the maintainer of this project is still active? I've opened a PR #86 in the end of march and did not get any response.

@postgred, are you looking for support for maintaining this repo?

**Comment by postgred** [comment]
You're right I'm not active. It would be great if I find a new maintainer.
If you want to help with the repo, tell me what you want to do with the project and send a new PR.
If everything fine with ideas and code I will give you permissions.

**Comment by lehn-etracker** [comment]
I think keeping the full responsibility would be too much for a start, but becoming a collaborator would be fine.
I'm using `zabbixapi` Gem within a corporate Chef LWRP to manage Zabbix-items and other objects. I like the clear architecture of the gem and think it's quite easy to keep it's API working for recent versions of zabbix.

**Comment by jrbeilke** [comment]
Checked out the PR and LGTM.

Unfortunately I'm no longer using the zabbixapi gem regularily so I've also had less time to contribute. Two of the three PR's seem ready to merge, but #85 still needs a bit of work before we can close that out and bring the gem up to date with support for Zabbix 3.4

**Comment by shoekstra** [comment]
Hi @lehn-etracker, are you using a cookbook from the supermarket to manage your clients?  If so, which?

I'm about to get back into Zabbix and use Chef, so would be super handy if someone had taken care of the LWRP stuff to manage hosts via the API.

**Comment by lehn-etracker** [comment]
@shoekstra We're using Chef, too. I wrote a new LWRP to manage only templates and discovery rules to assign templates. That way, you don't need to manage hosts manually. During the next weeks, I will start to prepare our corporate `zabbix_lwrp_ng` LWRP Cookbook to be published.

**Comment by postgred** [comment]
@lehn-etracker Ok, we will give you collaborator during next week.

If you want to release a new version, text me to telegram(https://t.me/aladmit) or email(aaleksandrov@express42.com), I will answer faster than on GitHub.

**Comment by aladmit** [comment]
@lehn-etracker I'm so sorry for the long silence. Is collaborating still interesting for you?

**Comment by gdubicki** [comment]
Thanks for making me a maintainer, @aladmit . :) Can you also add me as an owner of the gem in rubygems.org so I can release new versions? My login there is gdubicki.

**Comment by gdubicki** [comment]
Sorry, but please see #93.
